### PR TITLE
Adds oh yea.

### PR DESCRIPTION
**Pull request by Walter-Sellars**
Adds the counterpart to our ubiquitous oh no. command. Now with koolaid man gifs.

### CODE REVIEW

**Comment by MordecaiMalignatus** [comment]
Why is this the koolaid man and not Randy Savage?

File: bot.py [hunk 1]
```diff
@@ -169,6 +169,10 @@ def handle_message_create(self, d):
 			cmd = CommandEvent(d['channel_id'], d['author'], None, self)
 			self.commands['ohno'](cmd)
 			return
+		if content.casefold() == 'oh yes.':
```
Comment (raylu) [inline bot.py]:
```suggestion
		elif content.casefold() == 'oh yes.':
```

File: utils.py [hunk 2]
```diff
@@ -123,3 +123,7 @@ def weather(cmd):
 def ohno(cmd):
 	url = 'https://www.raylu.net/f/ohno/ohno%03d.png' % random.randint(1, 294)
 	cmd.reply('', {'image': {'url': url}})
+
+def ohyes(cmd):
+	url = 'https://www.raylu.net/f/ohyea/ohyea%03d.gif' % random.randint(1, 6)
```
Comment (raylu) [inline utils.py]: I don't think we'll ever hit 100 images
```suggestion
	url = 'https://www.raylu.net/f/ohyea/ohyea%02d.gif' % random.randint(1, 6)
```

where'd you get the ones you DMed me from anyway?

Comment (Walter-Sellars) [reply utils.py]: internet and cutting youtube vids to gifs. Works for me. I might go full adhd one day and fill to 99

**Comment by Walter-Sellars** [comment]
reviewed changes completed

**Comment by raylu** [comment]
merged as 63bb6101671a8c82af78e42d32268f24538c3f35

**Pull request closed without merging**